source/War.py

```diff
@@ -0,0 +1,24 @@
+import sys
+
+sys.path.append("C:/Users/Chris/SoftwareEng/CIS322_SP22")
+
+from source.CardGames import *
+
+def initializePlayers(p1, p2):
```

Comment:
Great work on the code! Since your task is singular and independent of the other tasks, the functions you created won't conflict or overlap those of the other tasks.